src/components/command-bar/commands/search/unified-search/components/unified-hits-container/helpers/gather-search-tabs-content.tsx

```diff
@@ -81,8 +84,12 @@ function getOtherTabsWithResults(
 		})
 }
 
-export function gatherSearchTabsContent(
-	rawHits: Hit[],
+/**
+ * Transform unified search results into tab data that the
+ * `UnifiedHitsContainer` component can render.
+ */
+export function gatherSearchTabsData(
```

Comment:
Slight refactor here to match some keys to make things easier, and to allow `gather-search-tabs-content` to accept a unified search results object rather than a single array of raw hits.